router/http_test.go

```diff
@@ -1005,6 +1004,7 @@ func (s *S) TestDefaultServerKeypair(c *C) {
 	assertGet(c, "https://"+l.TLSAddr, "foo.example.com", "2")
 }
 
+// TODO(bgentry): validate this in Postgres??
 func (s *S) TestCaseInsensitiveDomain(c *C) {
```

Comment:
So I'm not actually sure how I should go about doing this validation. There's many different places where it could be added, from the schema and Controller API all the way down to Postgres itself. Is there a convention for this sort of thing across the project?